storybook/.storybook/config.js

```diff
@@ -23,12 +26,19 @@ addParameters({
     { name: 'Kawai', value: '#f8d3e0' },
     { name: 'Light', value: '#f7f7f7' },
   ],
-  options: { theme },
+  options: {
+    enableShortcuts: false,
+    theme,
+  },
 });
 
 // automatically import all files ending in *.stories.js
 const req = require.context('../src/elements', true, /.stories.js$/);
 function loadStories() {
+  if (window.nuxeo.mock) {
```

Comment:
Not a huge fan of the approach here, pretty sure there's a way to hook into events (ie https://storybook.js.org/docs/addons/api/#usechannel) but I'm ok to move fwd for now.
worst case scenario, if we're not able to hook into story load / unload events we should be able to create a decorator and use it's mount / unmount (or attach / detach) callbacks to set up the mock.

Reply:
Been there, done that, but no success. 😓 
Anyway, a ticket was created to dedicate more time to this task. (https://jira.nuxeo.com/browse/ELEMENTS-1120)